pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[tool.scikit-build]
+cmake.minimum-version = "3.26.4"
+
+[build-system]
+requires = ["scikit-build-core"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "nvjitlink"
+version = "0.1.0"
```

Comment:
Should we fill in other information here? Things like authors etc? The full list of supported fields is documented [here](https://packaging.python.org/en/latest/specifications/declaring-project-metadata/#declaring-project-metadata), and you could look at some RAPIDS packages for examples.